deluge_client/client.py

```diff
@@ -156,9 +156,11 @@ def _receive_response(self, deluge_version, partial_data=b''):
         if msg_type == RPC_ERROR:
             if self.deluge_version == 2:
                 exception_type, exception_msg, _, traceback = data
+                # On deluge 2, exception arguments are sent as tuple
+                exception_msg = b', '.join(exception_msg)
             else:
                 exception_type, exception_msg, traceback = data[0]
-            exception = type(str(exception_type.decode('utf-8', 'ignore')), (Exception, ), {})
```

Comment:
I added this because otherwise it fails with Python2

Reply:
Oh right, gotcha. I'll restore